qt/scientific_interfaces/Direct/ALFAnalysisModel.cpp

```diff
@@ -20,6 +20,9 @@
 using namespace Mantid::API;
 
 namespace {
+auto &ADS = AnalysisDataService::Instance();
+
+std::string const FIT_BASE_NAME("__fit");
```

Comment:
This depends on the answer to my other question but I would suggest maybe a more specific name if we are keeping the workspaces in the ADS. Atleast with ALFView in the name maybe...